feature_tests/dart/lib/src/OptionInputStruct.g.dart

```diff
@@ -38,6 +36,15 @@ final class OptionInputStruct {
     return struct;
   }
 
+  factory OptionInputStruct({int? a, Rune? b, OptionEnum? c}) {
+    final result = _OptionInputStruct_default_ctor();
+    final dart = OptionInputStruct._fromFfi(result);
+    dart.a = a;
+    dart.b = b;
+    dart.c = c;
```

Comment:
issue: these statements discard all fields from `default_ctor` and replace them with potential `null`s.

Reply:
ah! Right. I'll fix this